Makefile

```diff
@@ -1,10 +1,10 @@
 TIX_FILE := $(shell stack path --local-hpc-root)/combined/custom/custom.tix
 
 lint:
-	hlint src
```

Comment:
If you rebase we’d need these to work for tests as well and it would be great if it could work for .x and .y files as well.